xhtml2pdf/parser.py

```diff
@@ -569,6 +569,9 @@ def pisaLoop(node, context, path=None, **kw):
             value = str(context.cssAttr["-pdf-keep-in-frame-mode"]).strip().lower()
             if value in ("shrink", "error", "overflow", "truncate"):
                 keepInFrameMode = value
+            else:
+                keepInFrameMode = "shrink"
```

Comment:
this doesn't make sense... `if value in ("shrink", "error", "overflow", "truncate")` <- it already checks for "shrink"